plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -262,7 +262,7 @@ public static void main(String[] args)
                         schema,
                         ImmutableMap.of(),
                         ImmutableMap.of("http-server.http.port", "8080"),
-                        ImmutableMap.of(),
+                        ImmutableMap.of("delta.enable-non-concurrent-writes", "true"),
```

Comment:
we may want to enable this on per-test basis, wdyt?

(revert or discuss)

Reply:
I will revert - it is leftover from some manual testing.